test/ably/realtimeconnection_test.py

```diff
@@ -330,5 +330,27 @@ def once_disconnected(state_change):
         assert state_change.current == ConnectionState.DISCONNECTED
         assert state_change.reason.code == 80003
         assert state_change.reason.status_code == 408
+        await ably.close()
+
```

Comment:
you should be able to make a test for RTN17e (have a look at the fallback rest tests using `respx` to intercept http calls)